pkgs/applications/misc/zk-shell/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub, pythonPackages }:
+
+buildPythonApplication rec {
+  version = "1.0.0";
+  name = "zk-shell-" + version;
+
+  src = fetchFromGitHub {
+    owner = "rgs1";
+    repo = "zk_shell";
+    rev = "a59979de4e0c3747378a88e93e79a76ba34d2458";
+    sha256 = "0zisvvlclsf4sdh7dpqcl1149xbxw6pi1aqcwjbqblgf8m4nm0c7";
+  };
+
+  propagatedBuildInputs = (with pythonPackages; [
+    ansi kazoo nose six tabulate twitter readline
+  ]);
+
+  doCheck = true;
```

Comment:
`doCheck` is implicit true for `buildPythonApplication`. 

But the test suite does not work for me, because no `zookeeper` instance is running: https://github.com/rgs1/zk_shell/blob/master/.travis.yml#L10

If you do not find an easy way to fix this, you can skip the test suite as well.